config/src/main/resources/org/springframework/security/config/spring-security-5.8.rnc

```diff
@@ -1298,6 +1298,9 @@ xss-protection.attlist &=
 xss-protection.attlist &=
 	## Add mode=block to the header or not, default is on.
 	attribute block {xsd:boolean}?
+xss-protection.attlist &=
+	## Specify the value for the X-Xss-Protection header. When set, overrides both enabled and block attributes.
+	attribute header-value {"0","1","1\;mode=block"}?
```

Comment:
My IDE (IntelliJ) complains about this line, with

    group of "string" or "data" element

Is this the correct change to make?

Reply:
Good catch, thanks!

Changed to `"0" | "1" | "1; mode=block"`, it generates a very similar XSD but IntelliJ doesn't complain.

Note, I am no RELAX NG expert. I "made it work" but I don't really understand how...